### PR TITLE
feat: integrate with Packit

### PR DESCRIPTION
**Pull request by runcom**

### CODE REVIEW

File: fido-device-onboard.spec [hunk 1]
```diff
@@ -41,20 +63,408 @@ BuildRequires: cryptsetup-devel
 BuildRequires: clang-devel
 BuildRequires: make
 BuildRequires: python3-docutils
+# List of bundled crate in vendor tarball, generated with:
+# cargo metadata --locked --format-version 1 | CRATE_NAME="greenboot" ./bundled-provides.jq
```
[the remark this answers is not in the document]
Comment (runcom) [reply fido-device-onboard.spec:67]: leftover, great catch :man_dancing:

File: .packit.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  trigger: pull_request
+  targets:
+  - centos-stream-9-aarch64
+  - centos-stream-9-x86_64
+  - fedora-development-aarch64
+  - fedora-development
+- job: copr_build
+  trigger: commit
+  branch: main
+  owner: "runcom" # copr repo namespace
```
Comment (7flying) [inline .packit.yaml:41]: TODO: change this, @nullr0ute or @miabbott might be able to help creating a fedora-iot or edge namespace

Comment (nullr0ute) [reply .packit.yaml:41]: Looking at it now

Comment (runcom) [reply .packit.yaml]: I've removed this piece for now until we have the copr group and the PR is ready so other PR can rebase and have their PR create RPMs too

Comment (nullr0ute) [reply .packit.yaml]: there's probably a shorter URL

**Comment by runcom** [comment]
follow up https://github.com/fedora-iot/fido-device-onboard-rs/pull/457

**Review by 7flying — APPROVED** [review]
LGTM thanks!

**Pull request merged**